e2e-tests/fixtures/application.js

```diff
@@ -139,6 +141,9 @@ const application = {
     [MAXIMUM_BUYER_WILL_OWE]: '500',
     [NEED_PRE_CREDIT_PERIOD]: 'false',
     [CREDIT_PERIOD_WITH_BUYER]: 'Mock description',
+    [ACCOUNT_NUMBER]: '123456',
+    [SORT_CODE]: '123456',
+    [BANK_ADDRESS]: 'Mock address',
```

Comment:
Can we wrap this in an `LOSS_PAYEE_FINANCIAL_UK` object

Also can we update the account number with maximium characters and have "Mock bank UK address" instead of "Mock address"? We have "Mock address" in a few places